index.html

```diff
@@ -291,7 +291,7 @@ <h3>General rules for exposing <abbr title="Accessible Rich Internet Application
+		  <p>Host languages can have features that have implicit <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> semantics corresponding to <a class="termref">roles</a>. When a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role is provided that has a corresponding role in the accessibility <abbr title="Application Programming Interface">API</abbr>, user agents MUST use the semantic of the <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role for processing, not the native semantic, unless the role requires <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> states and properties whose attributes are explicitly forbidden on the native element by the host language.  Values for roles do not conflict in the same way as values for states and properties, and because authors are expected to have a valid reason to provide a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role even on elements that would not normally be repurposed. For example, spin buttons are typically constructed from text fields (<code>&lt;input  type=&quot;text&quot;&gt;</code>) in order to get most of the default keyboard support.  But, the native role, &quot;text field&quot;, is not correct because it does not properly communicate the additional features of a spin button.  The author adds the <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role of <code>spinbutton</code> (<code>&lt;input type=&quot;text&quot; role=&quot;spinbutton&quot; ...&gt;</code>) so that the control is properly mapped in the accessibility <abbr title="Application Programming Interface">API</abbr>. When a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role is provided that does not have a corresponding role in the accessibility <abbr title="Application Programming Interface">API</abbr>, user agents MAY expose the native semantic in addition to the <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role. If the host language element is overridden by a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> role whose semantics or structure is not equivalent to the native host language semantics or to a subclass of those semantics, then treat any required owned elements of the native role as having role <a href="#role-map-presentation">presentation</a> or <a href="#role-map-none">none</a>.</p>
```

Comment:
I removed the reference `[=ARIA/owned=]` so that the we don't break the build (`owned` is no longer a defined term) but I am having trouble re-writing this sentence to not use `owned` -- because I do not understand it.

For ease of reading, see the section [Conflicts between native markup semantics and WAI-ARIA](https://w3c.github.io/core-aam/#mapping_conflicts), the last sentence before the note, copied here:

> If the host language element is overridden by a WAI-ARIA role whose semantics or structure is not equivalent to the native host language semantics or to a subclass of those semantics, then treat any required owned elements of the native role as having role [presentation](https://w3c.github.io/core-aam/#role-map-presentation) or [none](https://w3c.github.io/core-aam/#role-map-none).

Does this mean that if the `native role` (the role as defined in html-aam for the html element, ignoring and `role=...`) has `required accessibility children` but the overriding WAI-ARIA role in `role=....` does not have the same required accessibility children.... then the required accessibility children of the `native role` should be given `role=none`?

I can't come up with an example of this. Also, if there is a WAI-ARIA role define with `role=...`, then the required accessibility children are only for that role, not the "native role".

Maybe this means for the fall back case, you need to have the required accessibility children of the `native role`?

But what is the example of this??

Reply:
maybe @joanmarie as a previous editor remembers this sentence or @scottaohara can bring in some html-aam insight? :)

Reply:
Unrelated, but this line diff is making me really wish we broke up paragraphs into multiple code lines 😅

Reply:
yeah we are about to do that to all the spec, actually, @smhigley :)

Reply:
Its a long sentence but I think this is what it is trying to say:

If the host language element is overridden by a WAI-ARIA role whose semantics or structure is not equivalent to the native host language semantics or to a subclass of those semantics, then treat any child elements having roles specified as Allowed Accessibility Child Roles as having role presentation or none.